scripts/create.ts

```diff
@@ -29,37 +47,37 @@ const chainLocations = getDirectories(join(__dirname, "../src/chains")).map(
   d => `chains/${d}`
 );
 locations = locations.concat(chainLocations);
-const argv = yargs
-  .command(
-    `$0 <name> --location [--folder]`,
-    `Create a new package in the given location with the provided name.`,
-    yargs => {
-      return yargs
-        .usage(
-          chalk`{hex("#e4a663").bold Create a new package in the given {dim <}location{dim >} with the provided {dim <}name{dim >}.}\n\n` +
-            chalk`{bold Usage}\n  {bold $} {dim <}name{dim >} {dim --}location {dim <}location{dim >} {dim [--folder <folder>]}`
-        )
-        .positional("name", {
-          describe: `          The name for the new package.`,
-          type: "string",
-          demandOption: true
-        })
-        .option("location", {
-          alias: "l",
-          describe: `The location for the new package.`,
-          choices: locations,
-          demandOption: true
-        })
-        .option("folder", {
-          alias: "f",
-          describe: chalk`Optional override for the folder name for the package instead of using {dim <}name{dim >}.`
-        });
-    }
-  )
+
+const argv = yargs(getArgv())
+  .command(`${COMMAND_NAME} <name>`, "", yargs => {
+    return yargs
+      .usage(
+        chalk`{hex("#e4a663").bold Create a new package in the given {dim <}location{dim >} with the provided {dim <}name{dim >}.}\n\n` +
+          chalk`{bold Usage}\n  {bold $} ${COMMAND_NAME} {dim <}name{dim >} {dim --}location {dim <}location{dim >} {dim [--folder <folder>]}`
+      )
+      .positional("name", {
+        // the spaces here are a hack to make this command description line up with the others in the help output
+        describe: `          The name for the new package.`,
+        type: "string",
+        demandOption: true
+      })
+      .option("location", {
+        alias: "l",
+        describe: `The location for the new package.`,
+        choices: locations,
+        demandOption: true
+      })
+      .option("folder", {
+        alias: "f",
+        describe: chalk`Optional override for the folder name for the package instead of using {dim <}name{dim >}.`,
+        type: "string"
+      });
+  })
   .demandCommand()
   .version(false)
   .help(false)
   .updateStrings({
+    // a little hack just to join the "Positionals" section with the "Options" section, for brevity
     "Positionals:": chalk.bold("Options"),
     "Options:": ` `,
```

Comment:
There's absolutely no reason to have brevity here, but you seem persistent about keeping it so this will be my last objection to mixing different types of arguments in the help message